dotcom-rendering/src/components/ScrollableCarousel.tsx

```diff
@@ -51,10 +51,12 @@ const carouselContainerStyles = css`
 	${from.tablet} {
 		gap: ${space[2]}px;
 	}
-	${from.wide} {
+
+	/** this is breaking the big card at wide breakpoint carousel */
```

Comment:
I'll take a look at this when this PR is merged as there are a couple of spacing edge cases that need resolving.